.github/workflows/test_install.yml

```diff
@@ -0,0 +1,51 @@
+on: [push]
```

Comment:
maybe we should do this on a daily schedule; and crate an issue automatically if it fails. We have an example on the generator repo.